City_CRUD_API/schemas.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str
+
+
+class CityCreate(CityBase):
+    pass
+
+
+class CityList(CityBase):
+    id: int
+
+    class Config:
+        # from_attributes = True
+        orm_mode = True
+
+
+class CityDelete(CityBase):
+    id: int
```

Comment:
The `CityDelete` schema includes an `id` field, which might not be necessary if this schema is only used for deletion responses. Consider whether this schema is needed or if a simpler response model would suffice.